### ISSUE TITLE
IFTTT-Plugin description needs update

### ISSUE THREAD

**Issue by produnis** [friendica/friendica#3950]
Hi there,
today I managed to connect a foreign twitter.account with friendicas IFTTT-Plugin. All posts of the twitter account are mirrored as my own posts in friendica.
While setting it up, I noticed that some variablenames changed at ifttt.com.  Thus, IFTTT-Plugin description needs to be updated.
While on "`admin/plugins/ifttt`" everything is fine, the user settings at "`settings/connectors/IFTTT Mirror`" needs some changes:

> Create an account at IFTTT. Create three Facebook recipes that are connected with Maker (In the form 'if Facebook then Maker') with the following parameters:

should be changed into

> Create an account at IFTTT. Create three Facebook recipes that are connected with Webhooks (In the form 'if Twitter then Webhooks') with the following parameters:
(while the url for "Webhooks" is https://ifttt.com/maker_webhooks )

"Body for 'new status message'" should be:

> key=SUPERSECRET&type=status&msg=<<<{{Text}}>>>&date=<<<{{CreatedAt}}>>>&url=<<<{{LinkToTweet}}>>>

Body for 'new link post':

> key=SUPERSECRET&type=link&msg=<<<{{Text}}>>>&date=<<<{{CreatedAt}}>>>&link=<<<{{LinkToTweet}}>>>

One might add the message that "new link post" includes the pictures via OEmbed code.

Greetings from Germany
Joe

**Comment by MrPetovan** [comment]
Thanks for the follow-up from our conversation. IFTTT has different variable names depending on the “this” part of the applet, so it wouldn’t necessarily apply to Facebook as well.

I guess we need a full directory of applets suggestion to work with Friendica.

**Comment by produnis** [comment]
Does it mean that ifttt.com "knows" what $if-input I selected and then presents different variable-names in the Webhook-Applet, accordingly?

**Comment by MrPetovan** [comment]
Yes it does, you can check them out in the applet configuration by clicking on “Add Ingredient”

**Comment by produnis** [comment]
Aaahhh.. I see... maybe, there should be a hint on the description page, that variablenames might differ, and then give one example, for e.g. facebook or twitter.

**Comment by MrPetovan** [comment]
Yep, and RSS too!

**Comment by produnis** [comment]
maybe a list of accepted variable names that friendica uses are helpfull, too.
- type
- msg
- date
- link
- url ...
because these are always the same, and just need to be filled with a changing
`=<<<{{VAR}}>>>`

**Comment by MrPetovan** [comment]
I agree, I had to look in the source to come up with my old RSS imports!

**Comment by poVoq** [comment]
Is this plugin generic enough that it could also be used with a selfhosted NodeRed or such? I assume it is mostly exposing some webhooks?

**Comment by MrPetovan** [comment]
What's NodeRed?

**Comment by poVoq** [comment]
https://nodered.org/
A very nice and easy to self-host software like IFTTT.
There is also Huginn and some other IFTTT like software, but NodeRed is IMHO the best.

**Comment by MrPetovan** [comment]
So then yes, we support generic webhooks, the only important part is the user key you need to ass to the request to authenticate it.
